monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -62,7 +68,7 @@ def _get_ssh_struct(name: str, home_dir: str) -> Dict:
 
 
 def _get_ssh_files(
-    usr_info: Iterable[Dict], telemetry_messenger: ITelemetryMessenger
+    usr_info: Iterable[Dict], telemetry_messenger: ITelemetryMessenger, event_queue: IEventQueue
```

Comment:
One letter barely makes a difference.
```suggestion
    user_info: Iterable[Dict], telemetry_messenger: ITelemetryMessenger, event_queue: IEventQueue
```